src/App.jsx

```diff
@@ -0,0 +1,46 @@
+
+import {Route ,Routes} from 'react-router-dom'
+import {Home} from "./pages/home/Home.jsx"
+import { Builder } from './pages/builder/Builder';
+import Navbar from './components/organisms/Navbar';
+// import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+
+const App = () => {
+  const menuItems = [
```

Comment:
Can we please move the const declaration outside  the app function. 

It's better for performance since we won't mutate it, we don't need to re-declare the const every time the component renders.